mysql-test/std_data/dd/sdi/upgrade/mysql.json

```diff
@@ -23057,6 +23057,7 @@
         "schema_ref": "mysql",
         "se_private_id":NNN,
         "engine": "InnoDB",
+        "last_checked_for_upgrade_version_id": X,
```

Comment:
Is it upstream, or if it is not upstream, why we are different?

Reply:
This part is added by Satya https://github.com/percona/percona-server/commit/14e2a96c6954433970ce15b613dbbe85fb7239f0#diff-68a5096de97c85497eb7bb7ef339437b
but `last_checked_for_upgrade_version_id` was added in upstream 8.0.13